Woopra/Woopra-iOS
# Update doc and build config

### Purpose
1. Unify the product name using `WoopraSDK` and module name of `Woopra` for both `CocoaPods` and `SPM`.
2. Change version to `1.2.0`.
3. Rename the `.podspec` and update some details.
4. Update README.md to reflect the updates.

### How to Verify it
You can only verify it using SPM in Xcode:
1. Create or use an existing iOS App project.
2. Check if you can import the SDK by `import Woopra`.
- Using local pod for `CocoaPods`
```
pod 'WoopraSDK', :path => '/path/to/local/pod' #full path of the .podspec file.
```

- Using local Swift Package
1. Build the local SPM by executing `swift build`
2. Install the local SPM in your iOS project: Project > Package Dependencies > plus button > Add Local... > select the local project directory
![Screenshot 2024-04-24 at 12 42 53 PM](https://github.com/Woopra/Woopra-iOS/assets/10951814/8b3e380e-0d31-48d9-9af9-6c61e66a7323)
![Screenshot 2024-04-24 at 12 43 55 PM](https://github.com/Woopra/Woopra-iOS/assets/10951814/5955a132-06a1-4e24-a935-3ff9d247f449)
![Screenshot 2024-04-24 at 12 44 29 PM](https://github.com/Woopra/Woopra-iOS/assets/10951814/2fb621aa-c84f-48eb-9e30-0d7d4c0bb0f7)
![Screenshot 2024-04-24 at 12 44 57 PM](https://github.com/Woopra/Woopra-iOS/assets/10951814/cdc51092-dfcb-46f5-80a7-eecf0b2bff1b)

3. Visit Project > Package Dependencies again and verify that the package name is `WoopraSDK`
